test/typechecker/modal-counterexamples.t

```diff
@@ -3,3 +3,23 @@ Prawitz's example of failure of the substitution principle
   mtt: Type inference error: regular variable x (bound at 32:33) cannot accessed from boxed expression
        file name :  Not a file, lines :  0 - 0, column :  28 - 33
   [1]
+
+Error reports about an unknown regular variable in boxed expression in different cases:
+
+find a variable from the context behind a box
+  $ mtt infer -e "λx:A. box (λy:B. box (λz:[]A. y))"
+  mtt: Type inference error: regular variable y (bound at 30:31) cannot accessed from boxed expression
```

Comment:
"regular variable y (bound at 30:31)" -- that's not accurate, it's bound at 13:14.

So, in my opinion the error message should read something like this:
```
Type inference error: regular variable "y" (bound at <exact-place after the `λ`) is accessed in box expression (<range-of-the-box-expression>) at <the-place-inside-box-where-y-is-accessed>.
```

Reply:
You are right, but currently, we can't track the location of bindings. Except that it fixed. 

Also, I change the location of the error: error occurs not in whole box expression, but exact at variable.